tests/test_distributions.py

```diff
@@ -10,6 +10,9 @@
 from optuna import distributions
 
 
+_choices = (None, True, False, 0, 1, 0.0, 1.0, float("nan"), float("inf"), -float("inf"), "", "a")
```

Comment:
Note: `CategoricalChoiceType` is `Union[None, bool, int, float, str]`. `_choices` contains all types and their edge cases.